src/java.base/share/classes/java/io/PushbackReader.java

```diff
@@ -92,28 +93,14 @@ public int read() throws IOException {
     }
 
     /**
-     * Reads characters into a portion of an array.
-     *
-     * @param      cbuf  Destination buffer
-     * @param      off   Offset at which to start writing characters
-     * @param      len   Maximum number of characters to read
-     *
-     * @return     The number of characters read, or -1 if the end of the
-     *             stream has been reached
-     *
-     * @throws     IOException  If an I/O error occurs
-     * @throws     IndexOutOfBoundsException {@inheritDoc}
+     * {@inheritDoc}
      */
-    public int read(char cbuf[], int off, int len) throws IOException {
+    public int read(char[] cbuf, int off, int len) throws IOException {
```

Comment:
Shouldn't you add

```
 * @throws IndexOutOfBoundException {@inheritDoc}
 * @throws IOException {@inheritDoc}
```

here as well? IIRC the global {@inheritDoc} will not add them.

Reply:
No. In this case the specification no longer appears in the main method summary but rather under `Methods declared in class java.io.FilterReader` which has a full spec.